engines/director/score.cpp

```diff
@@ -101,6 +99,14 @@ Score::~Score() {
 			delete *it;
 
 	delete _labels;
+
+	if (_framesStream) {
```

Comment:
just similar to the lines above, drop the curly brackets, e.g.

```C++
if (_framesStream)
   delete _framesStream;
```

Do the same below.